src/deluge/gui/ui/keyboard/column_controls/velocity.cpp

```diff
@@ -57,6 +57,15 @@ bool VelocityColumn::handleVerticalEncoder(int8_t pad, int32_t offset) {
 	return false;
 };
 
+void VelocityColumn::handleLeavingColumn(ModelStackWithTimelineCounter* modelStackWithTimelineCounter,
+                                         KeyboardLayout* layout) {
+	if (!FlashStorage::keyboardFunctionsVelocityGlide) {
```

Comment:
I think we always want to reset to the held value, though I'm open to arguments in favor of this behavior.

Reply:
If the Glide is enabled there is no coming back to old value. Regardless of if you short press or long press the value is replaced. But if glide is disabled it must jump to the last short pressed pad

Reply:
Is there any way to grab the previous value when glide is enabled? If not we should add one even if just for the purpose of holding the top pad to change columns

Reply:
Ohh I know what you mean now. I think that if i remove the if condition here it will work as you want. I am going to test it...

Reply:
@m-m-adams @sapphire-arches  I made a change to remove the logic that was there to send the pad release event to the previous column that was there before changing the column type. And now the pad release is "forced" at the exact moment you change columns. Also removed the IF condition in velocity and modwheel so it also respect the last set velocity and modwheel when you leave those columns. That way, even in glide mode, you can set a lower velocity for example, hold the top pad to change the column type, and still have that low velocity set and not replaced by 127 because you held the top pad